tools/mixxx-isolate.py

```
@@ -0,0 +1,496 @@
+
+
+def startHelper(args):
+    """Execute $self with adjusted arguments under privilege elevation"""
+
+    helper = elevatePermissions(args)
+    mixxxProc = None
+    if not helper:
+        log.critical(" Could not spawn with elevated permissions")
+    while True:
+        try:
+            helper.poll()
+            if helper.returncode == 127:
+                # user aborted password request, nothing happened so it's
+                # save to exit
+                log.error(" Could not run elevate process. Aborting")
+                sys.exit(1)
+            elif helper.returncode == 23:
+                # Error setting up isolation
+                log.critical(" Error creating isolation. Aborting")
+                sys.exit(1)
+            elif helper.returncode is not None:
+                # helper process died
+                log.info(helper.stdout.read(-1))
+                log.error(
+                    " Helper process died: %s."
+                    " Cleanup might be required required",
+                    helper.returncode,
+                )
+                break
+
+            line = helper.stdout.readline().decode("utf-8", "ignore")
+            if line.startswith("STATUS:"):
+                status = line[7:].strip()
+                print("GOT STATUS:" + status)
+                if status == "ISOLATION":
+                    log.info(" Isolation setup complete. Starting mixxx...")
+                    mixxxProc = subprocess.Popen(
+                        [args.mixxx, "--engine-cpu-set", args.cpuset_engine]
+                        + args.mixxx_args
+                    )
+                    mixxxProc.wait()
+                    log.info(" Mixxx finished")
+                    helper.stdin.write("exit\n".encode("ASCII"))
+                    try:
+                        # this might likely not work
+                        helper.send_signal(signal.SIGUSR1)
+                    except PermissionError:
+                        pass
```

what is the value of silencing this exception?